README.md

```diff
@@ -25,31 +25,53 @@ User documentation is available at
 
 
 ## Setup for development
-Install Python version 3.8.8 or newer.
+Install Python version 3.8.8.
 The exact procedure depends on the operating system and configuration.
-Verify the version with
+
+
+### Prepare conda environment
+Assuming current working directory is `NiBAx` and containing the source code
+cloned from https://github.com/CBICA/NiBAx.git.
+
+Ensure Anaconda is installed. Follow instructions for user's operating system [here](https://docs.anaconda.com/anaconda/install/index.html). After Anaconda has been installed, be sure to exit and reopen any 
+command line windows to use `conda` command
+
+To create the enviroment:
+
+```shell
+conda create -n env python=3.8.8  
```

Comment:
Calling the environment `env` may be misleading. I think it is much better to call the environment `NiBAx`.